Probesys/bileto
# imp: List tickets where user is requester in "Your tickets"

## Related issue(s)



N/A

## Changes



- refactor TicketsSearcher to provide methods `getTicketsOfCurrentUser` and `getTicketsToAssign` (+ rename `count*` into `countTickets*`
- change the parameter to choose the index view from `assignee` to `view`
- change the mechanism of criteria in TicketRepository to handle `OR` requests on fields
- in `getTicketsOfCurrentUser`, change the assignee criterion by an assignee OR requester criterion

## How to test manually



- with a user, assign a ticket to yourself + set you as requester of another
- on the dashboard and in the "Your tickets" views (tickets indexes) → check you see both tickets

## Checklist
